db/migrate/20170815175201_swap_column_names.rb

```diff
@@ -0,0 +1,7 @@
+class SwapColumnNames < ActiveRecord::Migration
+  def change
+  	rename_column :scheduled_items, :admission, :admission_temp
+  	rename_column :scheduled_items, :audience, :admission
+  	rename_column :scheduled_items, :admission_temp, :audience
+  end
```

Comment:
I see. Then do you want me to research more that what I did here was the right method and comment my source? Or is it fine to leave it as is since it seems to work as intended on staging?